regimes/it/fpa_codes.go

```diff
@@ -0,0 +1,272 @@
+package it
+
+import (
+	"regexp"
+
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+	"github.com/invopop/gobl/i18n"
+)
+
+// FPACodeDefinition defines properties of an alphanumeric codes used by
+// FatturaPA, Italy's e-invoicing system. The codes are used to classify
+// various aspects of an invoice, namely the tax system, fund type, payment
+// method, document type, nature, and withholding type. An FPACode can include
+// uppercase letters, numbers, and a "." separator for the numeric portion.
+type FPACode string
+
+type FPACodeDefinition struct {
```

Comment:
@samlown I wanted to make use of the new `tax.KeyDefinition` from your other PR, but I found that defining a `Key` on top of `Code` seemed a bit too much for FPA codes.